### DISCUSSION TITLE
New play store version?

### DISCUSSION THREAD

**Discussion by NikunjK8** [celzero/rethink-app#1430] [Q&A] [answered]
The play store version had it's last update in April. It's a few versions behind now. Did you guys stopped releasing it on store?

**Comment by NikunjK8** [comment]
On a side note, can this app show which app made connection with which domain?

**Reply by ignoramous** [reply]
Yes. In *Network* logs.

**Reply by NikunjK8** [reply]
It's showing ip addresses but not domains. Tell me if I'm doing something wrong.

**Comment by ignoramous** [comment, marked as answer]
We haven't because of some curious bugs specific to Android 14 and are using releases on GitHub and F-Droid as a means to gather more bug reports around it.

Think the next version, `v055j` looks stable enough to release to the Play Store, which has around 500k users (by far, the largest base).
